lib/api/service/auth_service.dart

```diff
@@ -20,4 +25,10 @@ abstract class AuthServiceImpl extends AuthService {
   Future<AuthResponse> signIn(
     @Body() SignInRequest body,
   );
+
+  @override
+  @POST('/api/v1/oauth/token')
```

Comment:
@Wadeewee Did you interested in this ticket? 🙈 https://github.com/hoangnguyen92dn/survey-flutter-ic/issues/36

Reply:
I got the `401` on the first integration 🙈 That's why I need to include it. So, Can we include [#36](https://github.com/hoangnguyen92dn/survey-flutter-ic/issues/36) on this PR @hoangnguyen92dn

Reply:
@Wadeewee 🙆‍♂️ I will move #36 to current sprint